docker/compose/docker-compose-extendeds3.yml

```diff
@@ -18,6 +18,7 @@ services:
     image: pravega/bookkeeper
     ports:
       - "3181:3181"
+    restart: always
```

Comment:
What's the semantics of this?

Reply:
It specifies the "restart policy" for the container, which determines what happens when a container exits. It can take 3 values: `no`, `on-failure` or `always`. The default is `no`, meaning a container will not be started automatically. `on-failure[:max retry]` means that the container will restart only if it exits with non-zero status. We are using `always` for Bookkeeper container here, so it'll restart on exit.